WordPress/Classes/ViewRelated/Blog/SettingsMultiTextViewController.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
+
+@interface SettingsMultiTextViewController : UITableViewController
+
+@property (nonatomic, copy) void(^onValueChanged)(id);
+@property (nonatomic, copy) void(^onCancel)();
```

Comment:
Sorry @SergioEstevao, i didn't notice this last time. Is it possible to swich `id` with `NSString`?